lib/src/widgets/testable.dart

```diff
@@ -38,7 +38,10 @@ class TestableState extends State<Testable>
   static final Logger _logger = Logger('_TestableState');
 
   final List<StreamSubscription> _subscriptions = [];
-  final Set<TestableType> _types = {TestableType.tappable};
+  final Set<TestableType> _types = {
+    TestableType.long_pressable,
```

Comment:
Is there ever a use case when something is tappable but not long-pressable?  I can't think of one.  If there is one, I'm ok separating these types but if not, let's just keep them together and have both the tap step and the long press step key off of `tappable`.